cypress/support/commands.ts

```diff
@@ -1494,6 +1494,16 @@ export const createTaskFromEmpty = (
   cy.getByTestID('task-form-offset-input').type(offset)
 }
 
+export const disableClickThroughAnnouncement = () => {
```

Comment:
Is this change needed for OSS or otherwise safe for it (seems it is related to MQTT?)?